lib/test/t_FejerAlgorithm_std.cxx

```diff
@@ -0,0 +1,124 @@
+//                                               -*- C++ -*-
+/**
+ *  @brief The test file of class GaussKronrod
+ *
+ *  Copyright 2005-2020 Airbus-EDF-IMACS-ONERA-Phimeca
+ *
+ *  This library is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Lesser General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This library is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Lesser General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Lesser General Public License
+ *  along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+#include "openturns/OT.hxx"
+#include "openturns/OTtestcode.hxx"
+#include <cmath>
+
+using namespace OT;
+using namespace OT::Test;
+
+typedef Collection<Complex> ComplexCollection;
+
+int main(int, char *[])
+{
+  TESTPREAMBLE;
+  OStream fullprint(std::cout);
+  setRandomGenerator();
+
+  try
+  {
+
+    {
+      const SymbolicFunction f("x", "sin(x)");
+      const Scalar a = -2.5;
+      const Scalar b = 4.5;
+      // High-level interface
+      Collection<FejerAlgorithm> algos(0);
+
+      algos.add(FejerAlgorithm(Indices(1, 20), FejerAlgorithm::FEJERTYPE1) );
+      algos.add(FejerAlgorithm(Indices(1, 20), FejerAlgorithm::FEJERTYPE2));
+      algos.add(FejerAlgorithm(Indices(1, 20), FejerAlgorithm::CLENSHAWCURTIS));
+      FejerAlgorithm algo;
+      for (UnsignedInteger testCase = 0; testCase < algos.getSize(); ++ testCase)
+      {
+        algo = algos[testCase];
+        Scalar value = algo.integrate(f, Interval(a, b))[0];
+        const Scalar ref = cos(a) - cos(b);
+        assert_almost_equal(value, ref, 1e-10, 1e-10);
+        // Low-level interface
+        Sample adaptedNodes;
+        value = algo.integrateWithNodes(f, Interval(a, b), adaptedNodes)[0];
+        assert_almost_equal(value, ref, 1e-10, 1e-10);
+      }
+    }
+
+    {
+      Description inputs(2, "x");
+      inputs[1] = "y";
+      SymbolicFunction f(inputs, Description(1, "cos(pi_ * x / 2) * sin(pi_ * y)"));
```

Comment:
you use braces initiializers now:
```
SymbolicFunction f({"x", "y"}, {"cos(pi_ * x / 2) * sin(pi_ * y)"})
const Point lowerBounds = {-1.0, 0.0};

```

Reply:
Indeed. Code was done before your PR ;-)